e2e/deno-e2e/tests/deno.spec.ts

```diff
@@ -38,26 +38,23 @@ describe('Deno', () => {
 
   describe('application', () => {
     it('should create deno app', async () => {
-      await runNxCommandAsync(`generate @nrwl/deno:app ${appName}`);
+      // await runNxCommandAsync(`generate @nrwl/deno:app ${appName}`);
```

Comment:
Should this standalone (`@nrwl/deno:preset`) be in its own `describe`?

Or are we no longer testing `@nrwl/deno:app` in favour of `@nrwl/deno:preset`